mainWidget/main_widget.yaml

```diff
@@ -525,12 +543,12 @@ slots:
                                 default:
                                   - component: oh-repeater
                                     config:
-                                      fetchMetadata: metadata,semantics
-                                      filter: (loop.roomArray.metadata) && ( (loop.roomArray.metadata.semantics.value).includes("Room") || (loop.roomArray.metadata.semantics.value).includes("Corridor") )
+                                      fetchMetadata: metadata,semantics,widgetOrder
+                                      filter: (loop.roomArray.metadata) && ((loop.roomArray.metadata.semantics.value).includes("Room") || (loop.roomArray.metadata.semantics.value).includes("Corridor")) && ((loop.roomArray.metadata.semantics.config.isPartOf).includes(vars.objVar.floor))
                                       for: roomArray
                                       fragment: true
-                                      groupItem: =vars.objVar.floor
-                                      sourceType: itemsInGroup
+                                      itemTags: ","
+                                      sourceType: itemsWithTags
```

Comment:
I understand adding widgetOrder, but why do we need to check for isPartOf ? Checking group membership should be enough and speeds up the results, as repeater has to check onnly one groupd, but not all semantic items....

Reply:
Checked again, please revert this change back to itemsInGroup.

Reply:
With itemsInGroup an error occurs in the query of the restApi. This can be seen in the debugger console of the browser.
![grafik](https://user-images.githubusercontent.com/79790877/218311654-bb1d3686-0d3b-4d15-b1cd-b2b479c3c840.png)
If the objVar.floors is not initialized or empty (undefined) there is no item for the repeater.

When I change to itemsWithTags the error disappears.

The loading times are almost the same. With 5 measurements.
![grafik](https://user-images.githubusercontent.com/79790877/218312016-a4d752cf-07d0-4698-9e39-1144f6c8adb1.png)

Reply:
Ok, convinced 😉